job-descriptions/software-engineer.md

```diff
@@ -4,10 +4,12 @@
 
 ### About the company
 
-From sustainable energy systems to self-driving cars, the innovations of the future will all rely on code. With so much software to write, developers need a better way to access, understand, and collaborate on code.
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will increasingly rely on code, so if we can make software developers more productive, we can literally bring the future sooner.
 
-At Sourcegraph, we are building a better, smarter foundation for software development. One that will enable developers (and the companies that rely on them) to more quickly turn great ideas into groundbreaking technology. Effectively, we want to bring the future sooner.
-Check out what we are building for developers and why it matters: https://sourcegraph.com/plan.
+We have already built the best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large engineering organizations across the world.
+Our [browser extensions](https://about.sourcegraph.com/product/browser) provide code intelligence (e.g. hover tooltips, go to definition, find references) when viewing code in your code host.

```

Comment:
How about:
> [Sourcegraph](https://about.sourcegraph.com/product/server) and our [browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code. 

?

Reply:
"our" browser extensions is against the style guide, but I think it's okay here 🙂

Reply:
How about your sentence started with just "Sourcegraph's browser extensions..."?